presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/query/PinotAggregationProjectConverter.java

```diff
@@ -51,14 +51,28 @@
             "/", "DIV");
     private static final String FROM_UNIXTIME = "from_unixtime";
 
+    private static final Map<String, String> ARRAY_AGGREGATION_ALLOWED_PUSHDOWN_MAP = ImmutableMap.<String, String>builder()
```

Comment:
nit: PRESTO_TO_PINOT_ARRAY_AGGREGATIONS ?

Just something verbose to state what is it a map of (ie, what is the key/value).

Reply:
done.